lib/Ohmbrewer_Relay.cpp

```diff
@@ -44,6 +50,54 @@ Ohmbrewer::Relay::~Relay() {
     // Nothing to do here...
```

Comment:
What I mean is, this comment isn't necessary anymore because of what you added on lines 41 and 42 (i.e. you're doing something here).